ztoc/fbs/ztoc09.fbs

```diff
@@ -0,0 +1,50 @@
+namespace v09;
+
+table Xattr {
+	key : string;
+	value : string;
+}
+
+table FileMetadata {
+	name : string;
+	type : string;
+	uncompressed_offset : long;
+	uncompressed_size : long;
+	linkname : string;		// Target name of link (valid for TypeLink or TypeSymlink)
+	mode : long;			// Permission and mode bits
+	uid : uint32;			// User ID of owner
+	gid : uint32;			// Group ID of owner
+	uname : string;			// User name of owner
+	gname : string;			// Group name of owner
+
+	mod_time  : string;		// Modification time (is formatted in RFC 3339 format,
+							// with sub-second precision added if present: "2006-01-02T15:04:05.999999999Z07:00")
+	devmajor : long;		// Major device number (valid for TypeChar or TypeBlock)
+	devminor : long;		// Minor device number (valid for TypeChar or TypeBlock)
+
+	xattrs : [Xattr];
+}
+
+enum CompressionAlgorithm : byte { Gzip = 1 }
```

Comment:
Just want to confirm, later when we plan to add `zstd` later, changing this to 

```
enum CompressionAlgorithm : byte { Gzip = 1, Zstd = 2 }
```

won't be a breaking change right?

Reply:
Correct. That is not a breaking change.